postgres_mixin/alerts/postgres.libsonnet

```diff
@@ -315,7 +350,11 @@
               description: '{{ $labels.instance }} is requesting too many checkpoints, which may lead to performance degradation.',
               summary: 'PostgreSQL too many checkpoints requested.',
             },
-            expr: '\n              rate(pg_stat_bgwriter_checkpoints_timed_total{asserts_env!=""}[5m]) /\n              (rate(pg_stat_bgwriter_checkpoints_timed_total{asserts_env!=""}[5m]) + rate(pg_stat_bgwriter_checkpoints_req_total{asserts_env!=""}[5m]))\n              < 0.5',
+            expr: |||
+              rate(pg_stat_bgwriter_checkpoints_timed_total{}[5m]) /
+              (rate(pg_stat_bgwriter_checkpoints_timed_total{}[5m]) + rate(pg_stat_bgwriter_checkpoints_req_total{}[5m]))
+              < 0.5
+            |||,
```

Comment:
Again, description could do with more specificity here.

Also, am I reading this wrong in that if the percentage of checkpoints timed total vs total+reqd is LESS THAN 50% it's considered "too many?

Reply:
The expression calculates the ratio of timed checkpoints to the total number of checkpoints (both timed and requested).

Timed checkpoints are those that PostgreSQL performs based on a schedule
Requested checkpoints are typically triggered by certain conditions, such as reaching a specified number of WAL segments.

If this ratio drops below 0.5 (i.e., less than 50% of the checkpoints are timed), it implies that a higher proportion of checkpoints are being requested rather than occurring naturally based on the schedule.

Reply:
Gotcha thanks for explaining